src/orchard/spec.clj

```diff
@@ -4,31 +4,47 @@
    [clojure.string :as str]
    [clojure.walk :as walk]))
 
-(defmacro spec [fname & args]
-  `(when-let [f# (or (resolve (symbol "clojure.spec.alpha" ~fname))
-                     (resolve (symbol "clojure.spec" ~fname)))]
-     (f# ~@args)))
-
-(defmacro spec-gen [fname & args]
-  `(when-let [f# (or (resolve (symbol "clojure.spec.gen.alpha" ~fname))
-                     (resolve (symbol "clojure.spec.gen" ~fname)))]
-     (f# ~@args)))
+(defn- spec [sym & args]
+  (when-let [f (resolve sym)]
```

Comment:
Let's try `require`ing before `resolve`ing

Reply:
It's a bit more verbose though :/